opm/input/eclipse/Schedule/Schedule.hpp

```diff
@@ -462,6 +464,7 @@ namespace Opm
         std::vector<ScheduleState> snapshots;
         WriteRestartFileEvents restart_output;
         CompletedCells completed_cells;
+        std::shared_ptr<SimulatorUpdate> simUpdateFromPython;
```

Comment:
Somehow this new member in Schedule just to keep track of the wells changed in one PYACTION seems a bit unclean. I don't know whether there is another way, though.

Reply:
Yes, I'm not very happy with this either - I'd say the problem is that the Schedule needs to communicate something back to the Simulator, can we leave this for later?

Reply:
What shares the pointer?

Reply:
> What shares the pointer?

Sorry for the late reply: When a mocked simulator (https://github.com/OPM/opm-common/blob/bddd89d723dcc06fbe0443ee1210cd1dde35b975/msim/include/opm/msim/msim.hpp#L29) is constructed with a Schedule, the Schedule is copied.

@akva2 Do you say it's fine to leave this as a shared_ptr or shall this be changed to a unique_ptr?